source/documentation/platform/team-processes/feature-requests.html.md.erb

```diff
@@ -0,0 +1,45 @@
+---
+owner_slack: "#analytical-platform-notifications"
+title: Feature request process
+last_reviewed_on: 2024-06-20
+review_in: 6 months
+---
+
+# <%= current_page.data.title %>
+
+This document will explain how we respond to [feature requests](https://github.com/ministryofjustice/analytical-platform/issues/new?assignees=&labels=feature-request&projects=&template=feature-request-template.yml&title=+%E2%9C%A8+%3Ctitle%3E) that are created by our users and our team.
+
+## Requests to the team
+
+Requests can come in a few different paths the most common are;
+
+- Analytical Platform user raises [feature request](https://github.com/ministryofjustice/data-platform-support/issues/new/choose)
+- Discussion with user via email, slack where our team will ask if they can raise a formal request or do so on their behalf if for example they do not have an Analytical Platform account
+- Team members can raise [requests](https://github.com/ministryofjustice/analytical-platform/issues/new?assignees=&labels=feature-request&projects=&template=feature-request-template.yml&title=+%E2%9C%A8+%3Ctitle%3E) based on a [support issue](https://github.com/ministryofjustice/data-platform-support/issues) or engagement with users
+
+## Amigos reviews
+
+The Amigo's will review the [feature-request](https://github.com/ministryofjustice/analytical-platform/labels/feature-request) labelled issues and establish if there is enough information. If not, the Product Manager or Team will contact the requestor and gather more information
+
+## Prioritisation of the Feature Request
+
+- Once the necessary information is captured, the Amigos prioritised the requests based on the following
+- Criticality of request to the users of the Analytical Platform performing their duties
+Value the request will add to user experience of the platform
+Timelines of the requests (when it needs to be done)
+
+## Feature Request brought into the sprint
+
+- Issue gets refined by the team.
+- Once the requests are prioritised, the decision to bring that into the sprint is made.
+- The agreed plan is to use the original feature request ticket to deal with the request. Stories may be created by the team to create a new ticket where the story would be defined, making sure there is a definition of done etc, with additional issues linked to the feature request.
+
+![Diagram](images/feature-request-flow.excalidraw.png)
+
+## Amigos Team
+
+- Edwin Bajomo - Product Manager
```

Comment:
Given today is Edwins last day, is it worth updating to Darren?

Reply:
Or perhaps all those changes will happen together when Darren starts officially

Reply:
Is this table useful?

Reply:
Depends if the Amigos team is listed elsewhere - only needs to be in one place really